modules/custom-css/custom-css-4.7.php

```diff
@@ -149,6 +149,11 @@ public static function customizer_redirect() {
 	 * @return array $fields Modified array to include post_content_filtered.
 	 */
 	public static function _wp_post_revision_fields( $fields, $post ) {
+		// None of the fields in $post are required to be passed in this filter.
+		if( ! isset( $post['post_type'] ) || ! isset( $post['ID'] ) ) {
```

Comment:
Also, `isset()` can receive multiple parameters, and will only return true if all of them are set.  So probably simpler to use a single call instead.